tool/tctl/common/tctl.go

```diff
@@ -754,39 +758,69 @@ func (a *AuthCommand) GenerateAndSignKeys(client *auth.TunClient) error {
 		return trace.Wrap(err)
 	}
 
-	certPath := a.genUser + "-cert.pub"
-	keyPath := a.genUser
-	pubPath := a.genUser + ".pub"
-
-	// --out flag
-	if a.outDir != "" {
-		if !utils.IsDir(a.outDir) {
-			if err = os.MkdirAll(a.outDir, 0770); err != nil {
+	switch a.outputFormat {
+	//
+	// dump user identity into a single file:
+	//
+	case IdentityFormatFile:
```

Comment:
I'm not seeing the host certificate being exported and as far as I can tell the `--type` flag does not exist? Because the following from the issue does not work: `tctl auth sign --type=host`.